### DISCUSSION TITLE
Suggested workaround for breaking browser cache

### DISCUSSION THREAD

**Discussion by bmesuere** [observablehq/framework#929] [Q&A] [answered]
When I redeployed a Framework site using GitHub Pages, I noticed the old data was still being loaded. This probably was because of the browser caching the data file. Until #260 gets implemented, what is the suggested way to prevent this issue?

I now appened a dummy query string to the `FileAttachment()` call and this seems to work (I wasn't sure if the corresponding data loader would be triggered), but this solution is not very scalable.

Is there a better way to do this which I am not aware of?

**Comment by Fil** [comment, marked as answer]
your approach sounds ok, and the work on #260 is well advanced
